helpers/extension/driver.js

```diff
@@ -154,20 +154,23 @@ class ExtensionProtocol extends ChromeProtocol {
     });
   }
 
-  // Stubs to bypass a page reload for now.
-  /* eslint-disable no-unused-vars */
   off(eventName, cb) {
-    return undefined;
-  }
+    if (typeof this._listeners[eventName] === 'undefined') {
+      console.warn(`Unable to remove listener ${eventName}; no such listener found.`);
```

Comment:
we probably want to use the logging system, but that depends on #182 and implementing a npmlog replacement for the extension, so we can wait